base/server/src/main/java/org/dogtagpki/server/rest/UserService.java

```diff
@@ -976,9 +947,16 @@ public Response addUserCert(String userID, UserCertData userCertData) {
                 userCertData.setSubjectDN(cert.getSubjectDN().toString());
                 String certID = userCertData.getID();
 
-                userCertData = getUserCertData(userID, URLEncoder.encode(certID, "UTF-8"));
-
-                return createCreatedResponse(userCertData, userCertData.getLink().getHref());
+                userID = URLEncoder.encode(userID, "UTF-8");
```

Comment:
I think this line should be removed since the `userID` should not be encoded to call `getUserCertData()`.

Reply:
OK, done.